app/views/candidate_interface/continuous_applications_choices/index.html.erb

```diff
@@ -32,7 +32,12 @@
       <% else %>
         <% unless @application_form_presenter.can_add_more_choices? %>
           <p class="govuk-body">You cannot add any more applications.</p>
-          <p class="govuk-body">If one of your applications is unsuccessful, or you withdraw or remove it, you will be able to add another application.</p>
+          <p class="govuk-body">You can add more applications if:</p>
```

Comment:
Isn't this text contradictory: `You cannot add any more applications. You can add more applications if:` ?

What about `You may submit additional applications if:`?